CHANGELOG.md

```diff
@@ -1,20 +1,32 @@
 # Changelog
 
-## 9.2.0
+## Unreleased 
 
 ### Features
 
-- Add os and device attributes to Flutter logs ([#2978](https://github.com/getsentry/sentry-dart/pull/2978))
-- String templating for structured logs ([#3002](https://github.com/getsentry/sentry-dart/pull/3002))
-- Add user attributes to Dart/Flutter logs ([#3014](https://github.com/getsentry/sentry-dart/pull/3002))
 - SentryFeedbackWidget Improvements ([#2964](https://github.com/getsentry/sentry-dart/pull/2964))
- - Capture a device screenshot for feedback
- - Customize tests and required fields
- - Customization moved from the `SentryFeedbackWidget` constructor to `SentryFlutterOptions`:
+  - Capture a device screenshot for feedback
+  - Customize tests and required fields
+  - Customization moved from the `SentryFeedbackWidget` constructor to `SentryFlutterOptions`:
 ```dart
+// configure your feedback widget
 options.feedback.showBranding = false;
 ```
 
+### Dependencies
+
+- Upgrade Kotlin languageVersion to `1.8` ([#3032](https://github.com/getsentry/sentry-dart/pull/3032))
```

Comment:
Should we mark this a s a breaking change?

Reply:
I'll mark it as potentially breaking. the number of people potentially affected should be really low